yarn-project/builder/src/contract-interface-gen/typescript.ts

```diff
@@ -176,9 +177,11 @@ function generateArtifactGetter(name: string) {
  * @returns Code.
  */
 function generateAbiStatement(name: string, artifactImportPath: string) {
+  // TODO(MW): fix hack - AVM needs some public fns bytecode for ts testing
+  const loadPublic = name.includes('Avm') ? `Public` : ``;
   const stmts = [
     `import ${name}ContractArtifactJson from '${artifactImportPath}' assert { type: 'json' };`,
-    `export const ${name}ContractArtifact = loadContractArtifact(${name}ContractArtifactJson as NoirCompiledContract);`,
+    `export const ${name}ContractArtifact = load${loadPublic}ContractArtifact(${name}ContractArtifactJson as NoirCompiledContract);`,
   ];
   return stmts.join('\n');
 }
```

Comment:
Added a hack to force public function bytecode to be included for `ContractArtifacts` which are tested in avm ts. Later on I will probably add a method in the avm testing files to grab the bytecode rather than change how `noir-contracts` reads the artifacts.